Gopkg.lock

```diff
@@ -623,12 +623,14 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:8bc38ac4d30f7a54ee9c715a6ef3654c8b5e5a230b7719414177d8c57574f4cb"
+  digest = "1:e9eb895768e4f7b55b1e2978efa943f9bc4f66b5dfdc2d26f2e488ae5a3e1dce"
```

Comment:
Why is the version bump needed?

Reply:
This is just a `make dep` bump. I'll remove the vendor changes and use yaml for the testing.